docs/topics/hetzner-baremetal.md

```diff
@@ -305,6 +278,33 @@ sdd  disk <wwn>                  1.8T
 
 In the above case, you can use any of the four disks available to you on a baremetal server. 
 
+
+### Creating Workload Cluster 
+
+Since we have already created secret in hetzner robot, hcloud and ssh-keys as secret in management cluster, we can now apply the cluster.
```

Comment:
can you mention again that secrets and ssh key names are hard-coded and have to be named properly? You mention it now shortly above but there is no connection that it is hard-coded here in the template